README.md

```diff
@@ -25,37 +21,62 @@ Build Wasm and native code:
 cargo build --release
 ```
 
-## Run
+Great! You have already compile the Centrifuge Chain!
 
-### Tests
+## Tests
 
-```bash
-cargo test -p centrifuge-runtime --release
-```
+There are two kinds of tests, one related to how the *Centrifuge Chain* works itself
+and another one to verify how it works in a more real environment as a parachain.
 
-### Start local Relay chain(alice and bob) and Parachain(alice)
+### Chain tests
 
-Prerequisites:
-- [docker](https://docs.docker.com/get-docker/)
-- [*jd*](https://stedolan.github.io/jq/)
+The following command will run unitary and integration tests:
 
-Start relay chain
 ```bash
-./scripts/init.sh start-relay-chain
-```
-
-Start  centrifuge-chain as parachain
-```bash
-./scripts/init.sh start-parachain
+cargo test -p centrifuge-runtime --release
 ```
 
-Note: the command above will show logs and block until the parachain is stopped
-Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1`.
-
-Onboard parachain to Relay chain
-```bash
-./scripts/init.sh onboard-parachain
-```
+### Environment tests
+
+You can deploy a relay chain and connect a Centrifuge Chain node as parachain
+to it to verify how it behaves in the entire environment (end-to-end).
+
+0. Prerequisites. You must install these tools before:
+    - [docker](https://docs.docker.com/get-docker/)
+    - [*jd*](https://stedolan.github.io/jq/)
+
+1. Start a local [relay chain](https://wiki.polkadot.network/docs/learn-architecture#relay-chain).
+It contains two [validator](https://wiki.polkadot.network/docs/learn-validator) nodes
+    (Alice and Bob):
+    ```bash
+    ./scripts/init.sh start-relay-chain
+    ```
+    After a few seconds you can see the block production of the relay chain using the [polkadot.js (on localhost:9944)](https://polkadot.js.org/apps/?rpc=ws%3A%2F%2Flocalhost%3A9944#/explorer) client.
+
+    *Note: You can stop the relay chain using `./scripts/init.sh stop-relay-chain`*
+
+2. Start a *Centrifuge Chain* as [parachain](https://wiki.polkadot.network/docs/learn-parachains).
+It run a [collator](https://wiki.polkadot.network/docs/learn-collator) node:
+    ```bash
+    ./scripts/init.sh start-parachain
+    ```
+    *Note: the command above will show logs and block until the parachain is stopped.
+    Detailed logs may be shown by running the node with the following environment variables set: `RUST_LOG=debug RUST_BACKTRACE=1`.
```

Comment:
I have maintained this line. If I do not write RUST_LOG=debug, I can see the debug logs. Could it be outdated?

Reply:
Maybe 🤔 or maybe you have it set in your environment by default?

Reply:
Check it, I have not already set

Reply:
Ok, I have taken the liberty of removing the entire line in order to center the focus on "how to get the things working". Maybe later we could create a new section about *debugging* where to put things like how to enable backtrace and so on. Is it ok for you?

Reply:
That sounds really good to me :)